src/modules/inspect/token-viewer/token-viewer.ts

```diff
@@ -33,9 +33,9 @@ export class TokenViewer {
     this._inspectCorrelationService = inspectCorrelationService;
   }
 
-  public correlationChanged(): void {
-    const correlationWasInitiallyOpened: boolean = this.flowNode === undefined;
-    if (correlationWasInitiallyOpened) {
+  public processInstanceIdChanged(): void {
+    const processInstanceWasInitiallyOpened: boolean = this.flowNode === undefined;
```

Comment:
Maybe I'm missing something here, but from the name of this business rule, shouldn't this be something like

```suggestion
    const processInstanceWasInitiallyOpened: boolean = this.flowNode !== undefined;
```

?

Maybe find a better name?